src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -35,22 +35,23 @@ class HoldRequest extends React.Component {
    * submitRequest()
    * Client-side submit call.
    */
-  submitRequest(e, itemId) {
+  submitRequest(e, bibId, itemId) {
     e.preventDefault();
 
     axios
-      .get(`/api/newHold?id=${itemId}`)
+      .get(`/api/newHold?itemId=${itemId}`)
       .then(response => {
         if (response.data.error && response.data.error.status !== 200) {
-          this.context.router.push(`/hold/confirmation/${itemId}?errorMessage=` +
+          this.context.router.push(`/hold/confirmation/${bibId}-${itemId}?errorMessage=` +
```

Comment:
This looks nice. But I am wondering if `bibID-itemID` is the best way to show on the URL. I am thinking maybe `bibID/itemID` or even `?bibid=1234567` and `?itemid=7654321`. Thus it will be easier to extract bib id or item id for future use.